x-pack/plugins/security_solution/public/management/pages/policy/store/policy_details/selectors.ts

```diff
@@ -32,10 +32,26 @@ export const licensedPolicy: (
   licenseState,
   (policyData, license) => {
     if (policyData) {
-      unsetPolicyFeaturesAboveLicenseLevel(
+      const policyValue = unsetPolicyFeaturesAboveLicenseLevel(
```

Comment:
below, in a few places inside of this `if()` block, you it is using `policyData?` and `policyData?.inputs[0]?` - are the `?` really needed? Because `policyData` is already set to `PolicyData` type and we're inside of the `if()` that ensure its defined, I would have thought that TS would be happy :)

Reply:
ah you're righto!